build.sbt

```diff
@@ -31,13 +31,7 @@ lazy val microservice = Project(appName, file("."))
     testGrouping in IntegrationTest := TestPhases.oneForkedJvmPerTest((definedTests in IntegrationTest).value),
     parallelExecution in IntegrationTest := false
   )
-  .settings(
-    resolvers ++= Seq(
-      Resolver.bintrayRepo("hmrc", "releases"),
-      Resolver.jcenterRepo,
-      Resolver.bintrayRepo("emueller", "maven")
-    )
-  )
+  .settings(resolvers ++= Seq(Resolver.bintrayRepo("hmrc", "releases"), Resolver.jcenterRepo, Resolver.bintrayRepo("emueller", "maven")))
```

Comment:
Did you mean to change this file?  (I think the contents are the same but the fact that it's in this PR makes me twitchy).

Reply:
I merged the master to my branch & resolved the merge conflicts, which was how this was added in

Reply:
This is just style change. It looks exactly the same like before but in one line